mssql-ballerina/Client.bal

```diff
@@ -0,0 +1,222 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/crypto;
+import ballerina/jballerina.java;
+import ballerina/sql;
+
+# Represents a MsSQL database client.
+public client class Client {
+    *sql:Client;
+    private boolean clientActive = true;
+
+    # Initialize Mssql Client.
+    #
+    # + host - Hostname of the mssql server to be connected
+    # + instanceName - Instance of mssql server to connect to on serverName
+    # + user - If the mssql server is secured, the username to be used to connect to the mssql server
+    # + password - The password of provided username of the database
+    # + database - The name fo the database to be connected
+    # + port - Port number of the mssql server to be connected
+    # + options - The Database specific JDBC client properties
+    # + connectionPool - The `sql:ConnectionPool` object to be used within the jdbc client.
```

Comment:
input param order and doc comment fields order are not matched. we need to follow the name order

Reply:
Fixed with [`f03f003`](https://github.com/ballerina-platform/module-ballerinax-mssql/pull/2/commits/f03f003d8508393b56fd7e5653d1f40e473e3de9)